CHANGELOG.md

```diff
@@ -5,3 +5,7 @@ All notable changes to this project will be documented in this file.
 ## [0.1.0]() (2024-11-28)
 ### Features
 * Add AWS acm, alb, amplify, ecr, ecs-application, ecs-cluster, elasticache, iam-sso, identity-provider, opensearch, password-generator, postgres, route53, s3, sqs, ssm-parameter, static-website, vpc-endpoints, vpc, vpn modules.
+
+## [0.1.1]() (2024-11-29)
+### Features
+* Update postgres module
```

Comment:
please move it upward a @spartan-toannhan , the release pipeline retrieves the first tag as release tag.